web/ui/navbar/item.svelte

```diff
@@ -12,6 +13,7 @@
   export let secondary = false
   export let submenu = false
   export let small = false
+  export let role: AriaRole = 'menuitem'
```

Comment:
Where we plan to use this property?

I think adding options which we do not need make code readability worse. In my experience it is better to add option instead of inlined value only if we have plans to use option.

Reply:
I think during further re-factoring of role option we will need to pass sub role values to container element - I do not see other option to do that besides that to move it to properties

Reply:
What role do you plan to use in the future for the menu items?